zipkin-autoconfigure/collector-kafka10/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2017 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.zipkin.java</groupId>
+    <artifactId>zipkin-autoconfigure</artifactId>
+    <version>1.24.1-SNAPSHOT</version>
```

Comment:
I've rebased the kafka10-collector branch in my local repo and committed the version updates to these two pom files. It won't push cleanly, because my local branch and the one on github have diverged. I suspect I can force the push to resolve this (`git push -f origin +kafka10-collector'), but I'm wary of doing something that rewrites history and screws up the PR.

Am I heading in the right direction, or did I not execute the rebase properly?

Current state of affairs on my local clone of the repo:
```dgrabowski:zipkin$ git status
On branch kafka10-collector
Your branch and 'origin/kafka10-collector' have diverged,
and have 37 and 25 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
nothing to commit, working tree clean
dgrabowski:zipkin$ git push
To github.com:dgrabows/zipkin.git
 ! [rejected]        kafka10-collector -> kafka10-collector (non-fast-forward)
error: failed to push some refs to 'git@github.com:dgrabows/zipkin.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
dgrabowski:zipkin$```